### DISCUSSION TITLE
For external PR's, the workflow runs but does not show up on a PR for approval

### DISCUSSION THREAD

**Discussion by sayaliM0412** [community/community#136507] [Actions]
### Select Topic Area

General

### Body

**Issue**

- For external/forked PR's, the workflow shows up in "Action" tab for approval but does not show up on the PR page for approval.
- **workflow**
   - https://github.com/liquibase/liquibase/blob/master/.github/workflows/build-branch.yml
- **example Action** : 
   - https://github.com/liquibase/liquibase/actions/workflows/build-branch.yml
- **example PR**:
   - https://github.com/liquibase/liquibase/pull/6236

**Comment by Thesirix** [comment]
it's a known behavior on GitHub.

When a workflow is triggered by a PR from a forked repository, GitHub, for security reasons, does not automatically run these workflows. A maintainer needs to approve the workflow manually via the Actions tab of the main repository. This means that the approval must be done from this tab rather than directly on the PR page. If the workflow doesn't show up for approval on the PR page, it might be related to how the triggering event for the workflow is configured. It needs to be specifically set up for pull_request events rather than push events​

If you don’t see the "Approve and run" button on the PR page, also check that your workflow is properly configured to trigger on pull_request events. Sometimes, this button might not appear if the workflow is misconfigured or if the repository's permissions don’t allow for this action​

**Reply by sayaliM0412** [reply]
@Thesirix 
I have a similar workflow, that does show up on the PR for approval
- **workflow**
   - https://github.com/liquibase/liquibase/blob/master/.github/workflows/run-tests.yml
- **example Action** : 
   - https://github.com/liquibase/liquibase/actions/workflows/run-tests.yml
- **example PR**:
   - https://github.com/liquibase/liquibase/pull/6236
- **screenshot**: 
<img width="878" alt="Screenshot 2024-08-22 at 10 02 03 AM" src="https://github.com/user-attachments/assets/4d6ebbfa-8d2e-49f5-87b4-5e5bed004013">
